### DISCUSSION TITLE
Introducing logic to show the sentences

### DISCUSSION THREAD

**Discussion by bajjar12** [dermatologist/nlp-qrmine#92] [Ideas]
Being a qualitative researcher, I find QRmine extremely beneficial. While running the package, I realized that the output categories are just random words with assigned weights. If the data involves short transcripts, this is good, however, if the data involves long 10/15 pages transcripts such as interview data, then making sense of these words is difficult. I propose that the output should also show the complete sentence that particular "code" is in. This way I can get more insights into the relevant themes.

**Comment by dermatologist** [comment]
A sentence-wise word matching should work. If it doesn't sentence -> to_bag_of_terms and look for matches.
Implement in: https://github.com/dermatologist/nlp-qrmine/blob/develop/src/qrmine/nlp_qrmine.py
